### PR TITLE
Update Gitlab example for Trivy 0.2.0

### LINKED ISSUE #267: gitlab-ci.yaml template is incorrect
Hi, 

The gitlab template on the readme is incorrect and will not validate. There is no directories in gitlab-ci.yaml 

I think this is probably more inline with normal gitlab-ci.yaml

```
stages:
  - test

trivy:
  stage: test
  image: docker:stable
  services:
    - name: docker:dind
      entrypoint: ["env", "-u", "DOCKER_HOST"]
      command: ["dockerd-entrypoint.sh"]
  variables:
    DOCKER_HOST: tcp://docker:2375/
    DOCKER_DRIVER: overlay2
    # See https://github.com/docker-library/docker/pull/166
    DOCKER_TLS_CERTDIR: ""
  before_script:
    - apk add --no-cache curl
    - export VERSION=$(curl --silent "https://api.github.com/repos/aquasecurity/trivy/releases/latest" | grep '"tag_name":' | sed -E 's/.*"v([^"]+)".*/\1/')
    - echo $VERSION
    - wget https://github.com/aquasecurity/trivy/releases/download/v${VERSION}/trivy_${VERSION}_Linux-64bit.tar.gz
    - tar zxvf trivy_${VERSION}_Linux-64bit.tar.gz
  variables:
    DOCKER_DRIVER: overlay2
  allow_failure: true
  services:
    - docker:stable-dind
  script:
    - ./trivy --exit-code 0 --cache-dir $CI_PROJECT_DIR/.trivycache/ --no-progress --severity HIGH DOCKER_IMAGE
    #- ./trivy --exit-code 1 --severity CRITICAL  --no-progress DOCKER_IMAGE
  cache:
    paths:
      - $CI_PROJECT_DIR/.trivycache/

```

The above example is working and is tested.

### PR DESCRIPTION
**Pull request by webmutation**
fixes #267

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1187,11 +1197,11 @@ trivy:
   services:
     - docker:stable-dind
   script:
-    - ./trivy --exit-code 0 --severity HIGH --no-progress trivy-ci-test:${CI_COMMIT_REF_NAME}
-    - ./trivy --exit-code 1 --severity CRITICAL --no-progress trivy-ci-test:${CI_COMMIT_REF_NAME}
+    - ./trivy --exit-code 0 --cache-dir $CI_PROJECT_DIR/.trivycache/ --no-progress --severity HIGH DOCKER_IMAGE
+    #- ./trivy --exit-code 1 --severity CRITICAL  --no-progress DOCKER_IMAGE
```
Comment (knqyf263) [inline README.md]: Why is this line commented out?

File: README.md [hunk 2]
```diff
@@ -1187,11 +1197,11 @@ trivy:
   services:
     - docker:stable-dind
   script:
-    - ./trivy --exit-code 0 --severity HIGH --no-progress trivy-ci-test:${CI_COMMIT_REF_NAME}
-    - ./trivy --exit-code 1 --severity CRITICAL --no-progress trivy-ci-test:${CI_COMMIT_REF_NAME}
+    - ./trivy --exit-code 0 --cache-dir $CI_PROJECT_DIR/.trivycache/ --no-progress --severity HIGH DOCKER_IMAGE
```
Comment (knqyf263) [inline README.md]: In this case, it is expected that an image is already pushed to a registry?

Comment (webmutation) [reply README.md]: Indeed, the image is already pushed in a different stage.

File: README.md (hunk 1 above)
Comment (webmutation) [reply README.md]: I was running the example, cp error

Comment (knqyf263) [reply README.md]: So, would you fix it as well?

Comment (webmutation) [reply README.md]: yes, I will fix it

**Comment by knqyf263** [comment]
Thanks!

**Pull request merged**